### PR TITLE
Simplified Chinese has been added to many parts of the project, and README.md has been modified

### PR DESCRIPTION
**Pull request by rin0chan**

### CODE REVIEW

**Comment by marioortizmanero** [comment]
Wow that is actually awesome. You're from the same university as me so I take it you know me? I can't remember seeing your profile in the past. I'm very thankful for your contribution, if you want to help on other parts of Vidify let me know! If you want to chat, I recently made a [Discord server](https://discord.com/invite/yfJSyPv) :)

I don't know a thing of chinese so I'm not sure how good the translation is. Are you native? Or did you just use an online translator?

Give me a bit of time and I'll review your changes. Or is this not ready yet?

Edit: Ah I've just seen the "待翻译中" files, so I imagine you're still at it. Let me know when you're done :+1:

**Comment by totoneei** [comment]
> Wow that is actually awesome. You're from the same university as me so I take it you know me? I can't remember seeing your profile in the past. I'm very thankful for your contribution, if you want to help on other parts of Vidify let me know! If you want to chat, I recently made a [Discord server](https://discord.com/invite/yfJSyPv) :)
> 
> I don't know a thing of chinese so I'm not sure how good the translation is. Are you native? Or did you just use an online translator?
> 
> Give me a bit of time and I'll review your changes. Or is this not ready yet?
> 
> Edit: Ah I've just seen the "待翻译中" files, so I imagine you're still at it. Let me know when you're done 👍

Yes ahh, I'm from China, and I'm a freshman at the University of Zaragoza,and  I translated all those documents myself, "待翻译中" "this file is just something I created first. I'll translate it all when I have time.

**Comment by marioortizmanero** [comment]
I've fixed a few things I noticed:

* Let's not move anything into directories. I know it would be great, but it makes it super hard to review this. I'll fix it afterwards.
* I realized that there are lots of unnecessary files in `data` and such. I'll remove them after this PR, sorry if they were confusing.
* Please remove the files in `content` that aren't translated yet instead of having them with the "待翻译中" message. It'll probably confuse people, and it's best to just not have them until they're done.
* For some reason chinese blog entries appear in the main page? Do check it out with `hugo build`
* The chinese page doesn't work at all? Clicking on the language in the dropdown menu goes to a broken link. Check it out with `hugo build` as well.

I won't be able to work on this anymore today, I'll be back at a different time.

File: i18n/zh-CN.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+[home]
+other = "主页"
+[wiki]
+other = "Wiki"
+[blog]
+other = "Blog" # Missing
```
Comment (marioortizmanero) [inline i18n/zh-CN.toml:6]: This is missing as well

File: i18n/zh-CN.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+[home]
+other = "主页"
+[wiki]
+other = "Wiki"
+[blog]
+other = "Blog" # Missing
+[github]
+other = "Github"
+[contact]
+other = "Contact" # Missing
```
Comment (marioortizmanero) [inline i18n/zh-CN.toml:10]: This is missing as well

**Pull request closed without merging**